src/Gelf/Message.php

```diff
@@ -297,6 +297,15 @@ public function toArray()
         }
 
         // return after filtering false, null and empty strings
-        return array_filter($message, 'strlen');
+        return array_filter($message, array($this, 'filter'));
+    }
+
+    /**
```

Comment:
Can you provide a a short doc-code summary for the method? Maybe even move the contents from the inline comment in L300 to the Doccode?

Alternative suggestion: Pass the filter to `array_filter` as a closure - a class-method is probably over-the-top - it's not used anywhere else.